test/unit/math/mix/core/operator_divide_equal_test.cpp

```diff
@@ -1,15 +1,10 @@
-#include <test/unit/math/mix/mat/util/autodiff_tester.hpp>
+#include <test/unit/math/test_ad.hpp>
 
-struct op_divide_equal_f {
-  template <typename T1, typename T2>
-  static typename boost::math::tools::promote_args<T1, T2>::type apply(
-      const T1& x1, const T2& x2) {
-    typename boost::math::tools::promote_args<T1, T2>::type y = x1;
+TEST(mathMixCore, opratorDivideEqual) {
+  auto f = [](const auto& x1, const auto& x2) {
+    decltype(x1 + x2) y = x1;
```

Comment:
Without the `decltype`, it wouldn't work if `x1` is double and `x2` a `var`, because the `auto y` would infer `double` and then `y /= x2` wouldn't  compile.  There might be an easier way to write this.